app/design/frontend/base/default/template/sendit/checkout/onepage/shipping_method/available.phtml

```diff
@@ -92,9 +92,20 @@ $bliskapaczkaCodCode = Sendit_Bliskapaczka_Model_Carrier_Bliskapaczka::SHIPPING_
                             <?php endif; ?>
 
                             <?php if ($_rate->getCode() == $bliskapaczkaCode || $_rate->getCode() == $bliskapaczkaCodCode) { ?>
+                                <?php
```

Comment:
@pawelkarbowniczek To powinno znaleźć się w bloku. Zobacz, że foreach wykonujesz mapowianie, czyli kilka razy liczysz to samo. Dodatkowo dodłeś logikę do templatce a to nie jest nalepsze podejście, lepiej zrobić to w bloku.